src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,37 @@
+.catalog {
+  display: grid;
+  gap: 48px;
+
+  justify-content: center;
+
+  grid-template-columns: $widthCard;
+  grid-auto-rows: 300px;
+
+  @each $minWidth, $col in $screensMinWidth {
+    @media (min-width: $minWidth) {
+      grid-template-columns: repeat($col, $widthCard);
+    }
+  }
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+
+    /* stylelint-disable */
```

Comment:
why disabled linter?

Reply:
It wouldn't accept the indented multi-line values, I thought it would be easier to read the values like that. Will convert to 1-line! Thank you! :rocket: